docs/data_quality.md

```diff
@@ -0,0 +1,114 @@
+# Properties of a High Quality OSV Record
+
+## Version
+
+1.0.0 (SEMVER)
+
+## Purpose
+
+Describe the “good enough” OSV record that will be imported by OSV.dev
+
+### Out of scope
+
+This does not discuss the problem of record bit rot over time, after initial successful import. The problem of continuous revalidation and treatment of records that have been successfully imported will be dealt with separately in .
+
+Deferred to a future iteration: validating the existence of vulnerable functions in the `ecosystem_specific` field, if supplied.
+
+## Audience
+
+1. Current and aspiring OSV record producers
+2. Downstream OSV.dev record consumers
+
+## Rationale
+
+OSV.dev seeks to be an comprehensive, accurate and timely database of known vulnerabilities (that is highly automation friendly). In order to meet this accuracy goal, a quality bar needs to be both defined and sustainably enforced.
+
+## Properties of a High Quality OSV Record
+
+### Valid
+
+As a prerequisite, it is assumed that a record passes [JSON Schema validation](#appendix-a-osv-schema-validation) for the version of the OSV Schema it declares itself to comply with in the `schema_version` field, or 1.0.0 if it does not.
+
+### Precise
+
+A high quality OSV record allows a consumer of that record to be able to answer the following questions in an **automated** way, at scale:
+
+* “Does this vulnerability, as described, impact me?
+  * “What version do I need to upgrade to for it not to impact me?” 
+
+The definition of “impact” will vary depending on how fine-grained the information available is (i.e. package-level or symbol-level for software library packages). Package-level precision is the minimum standard.
+
+#### Properties
+
```

Comment:
Forgive this nit, but the markdown linter in me demands that I at least suggest it

Both the `Precise` and `Identifiable` sections include `Property` subheadings, while their sibling `Valid` does not. Most markdown linters will complain if you have two headings with the same values, even if they are at different levels or are children of different parents. I would argue we don't need the the subheadings at all since the parent section of `Precise`, `Identifiable`, and `Property` already includes the word: `Properties of a High Quality OSV Record`

```suggestion
```

Reply:
I salute your pedantry as a fellow pedant 😄